packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -985,6 +1054,15 @@ export class UmbrellaStore<M extends BaseMetadata> {
       shallow
     );
 
+    const threadSubscriptions = DerivedSignal.from(
+      threadifications,
+      this.subscriptions.signal,
+      (t, s) => ({
+        subscriptions: s,
+        ...t,
+      })
+    );
```

Comment:
We don't have generic subscriptions yet, only `thread`s, and the only entry points for them are threads/inbox notifications endpoints so we're piggy-backing on `threadifications` changes.